### PR TITLE
[FIX] API-755 Deleted Bad File State

### PR DESCRIPTION
**Pull request by huarngpa**
* Make sheepdog change the file_state to deleted
* Fix problems with the deps

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -11,6 +11,7 @@ fuzzywuzzy==0.6.1
 graphene==0.10.2
 jsonschema==2.5.1
 lxml==3.8.0
+pyasn1==0.4.2
```
Comment (anmaxvl) [inline requirements.txt]: is there a reason it's moved from `dev-requirements` here?

Comment (huarngpa) [reply requirements.txt]: right, so the strategy here is that i'm trying to get rid of the `pyasn1-modules` in the dev-requirements. it causes ssls on my linux machine but for some reason magically works on our travis' instance. I'd rather let the cirrus and fence deps figure it out

Comment (anmaxvl) [reply requirements.txt]: are we using `cirrus` anywhere? that's something new.

Comment (huarngpa) [reply requirements.txt]: it's a dep for `fence` which is apparently getting used in conftest to generate a signed token

File: dev-requirements.txt [hunk 2]
```diff
@@ -28,8 +28,8 @@ SQLAlchemy==0.9.9
 -e git+https://github.com/uc-cdis/dosclient.git@1.0.0#egg=dosclient
 
 # these two are required by fence -> cirrus -> google_cloud
-pyasn1-modules==0.0.11
-urllib3==1.22
+# pyasn1-modules==0.0.11
+# urllib3==1.22
```
Comment (scintillating7) [inline dev-requirements.txt]: can we clean up this file (commented out lines + now out of date comment above)?

**Pull request merged**